src/utilities/report-utils.ts

```diff
@@ -2,8 +2,33 @@ import { queryValue } from "../utilities/url-query";
 import { getPortalData } from "../firebase-db";
 import { IPortalData, firebaseAppName } from "../portal-api";
 
-// TODO: switch default to production report version before production deploy
-export const DEFAULT_PORTAL_REPORT_URL = "https://portal-report.concord.org/branch/master/index.html";
+export const kProductionPortalReportUrl = "https://portal-report.concord.org/version/v4.1.0/index.html";
+export const kDevPortalReportUrl = "https://portal-report.concord.org/branch/master/index.html";
+
+// The default portal report URL is based on the URL:
+// - https://activity-player.concord.org and https://activity-player.concord.org/version/*
+//   defaults to the kProductionPortalReportUrl above
+// - everything else defaults to kDevPortalReportUrl which is should be master
+//
+// The default can be overridden with a portalReport URL param
+export const portalReportBaseUrl= ():string => {
+  const portalReportUrlParam = queryValue("portalReport");
+  if (portalReportUrlParam) {
+    return portalReportUrlParam;
+  }
+
+  const { origin, pathname } = window.location;
+  // According to the spec an empty path like https://activity-player.concord.org
+  // will still have a pathname of "/", but just to be safe this checks for the
+  // falsey pathname
+  if(origin === "https://activity-player.concord.org" &&
```

Comment:
Maybe move the url here to the top of the file as a constant?

Reply:
Done. 

I was tempted to go further with this but stopped myself.
portal-api.ts also uses this approach of looking at the URL, so they could share the constant. But even better there could be some shared functions for determining if this is a root or version url. But I didn't want to change all of that code while trying to get this released today. :)